src/color.rs

```diff
@@ -991,6 +1138,51 @@ mod tests {
             ((0.0, 0.0, 0.0), (0.0, 0.0, 0.0)),
         ]
     }
+
+    #[test]
+    fn ensure_color_mix() {
+        let mix_1 = Color::rgba(18., 55., 125., 1.0);
+        let mix_2 = Color::rgba(125., 33., 200., 0.8);
+        let expected = Color::rgba(72., 44., 163., 0.88);
+        let mix_res = mix_1.mix(&mix_2, 40);
+        assert_eq!(expected, mix_res);
+    }
+
+    #[test]
+    #[should_panic(expected = "101 is not a valid value for weight. Must be within 0-100 inclusive")]
+    fn invalid_mix_weight() {
+        let c1 = Color::rgb(1., 1., 1.);
+        let _ = c1.mix(&Color::rgb(2., 2., 2.), 101);
+    }
+
+    #[test]
+    fn check_rgb_to_hsl() {
+        let c = Color::rgb(251., 206., 33.);
+        let result = c.to_hsl();
+
+        assert_eq!((48., 0.9646017699115044, 0.5568627450980392), result);
+    }
+
+    #[test]
+    fn check_color_hue() {
+        let c = Color::rgb(250., 206., 33.);
```

Comment:
Could you use `Color::hsl` for these kinds of tests? I think it would make it more obvious that the asserted answer is actually correct. Please do that in this test and the ones below which test `with_hue`.